### ISSUE TITLE
sort the result of cheat --list by alphabet order

### ISSUE THREAD

**Issue by leslie-qiwa** [martialblog/cheatsheet#7]
This is current result

```$ cheat --list           │
 docker                                                                         │
 git                                                                            │
 markdown                                                                       │
 nc                                                                             │
 firefox                                                                        │
 pdb                                                                            │
 bash                                                                           │
 tmux-conf                                                                      │
 vagrant                                                                        │
 i3                                                                             │
 emacs                                                                          │
 tmux                                                                           │
 systemd

**Comment by martialblog** [comment]
Will be fixed in v2.0

https://github.com/martialblog/cheatsheet/tree/refactor

**Comment by martialblog** [comment]
Implemented in https://github.com/martialblog/cheatsheet/commit/23cd61887574350060d808d01877509300e9fce0
